awx/main/dispatch/worker/base.py

```diff
@@ -130,12 +130,16 @@ def run(self, *args, **kwargs):
         super(AWXConsumerPG, self).run(*args, **kwargs)
 
         logger.warn(f"Running worker {self.name} listening to queues {self.queues}")
+        init = False
 
         while True:
             try:
                 with pg_bus_conn() as conn:
                     for queue in self.queues:
                         conn.listen(queue)
+                    if init is False:
```

Comment:
Chatted w/ @chrismeyersfsu, it was necessary to swap around when we called this so that the postgres listeners are actually set up first.